recirq/qml_lfe/learn_dynamics_C.py

```diff
@@ -0,0 +1,167 @@
+import os
```

Comment:
what does the `_C` mean in this filename?

Reply:
The C stands for the "classical strategy" we outline in the paper.